Feature/homepage carousel

gabros20 opened the pull request.

liamsi commented on src/datas/glossary/data.js:
```diff
@@ -121,6 +121,11 @@ export const glossaries = [
+            {
+                url: 'pay-for-data',
+                title: 'PayForData',
+                text: 'A type of transaction, submitting a payload of arbitrary bytes (like a serialized block) , paying for the amount of block space that the submitted data takes up.'
```
```suggestion
                text: 'A type of transaction, submitting a payload of arbitrary bytes (like a serialized block), paying for the amount of block space that the submitted data takes up.'

```

liamsi commented on src/pages/glossary/pay-for-data.js:
```diff
@@ -0,0 +1,51 @@
+    render() {
+        return (
+            <div className={'glossary-content'}>
+                <p>A type of transaction, submitting a payload of arbitrary bytes (like a serialized block) , paying for the amount of block space that the submitted data takes up.</p>
```
```suggestion
                <p>A type of transaction, submitting a payload of arbitrary bytes (like a serialized block), paying for the amount of block space that the submitted data takes up.</p>
```

gabros20 committed:
Added initial carousel build

gabros20 committed:
Added new carousel component under hero section

gabros20 committed:
Updated card link styles

alex-beckett committed:
Add files via upload

alex-beckett committed:
Update apps-on-celestia.js

gabros20 committed:
Updated carousel design, and logic

gabros20 committed:
Merge branch 'main' into feature/homepage-carousel

gabros20 committed:
Updated carousel layout fixed images

gabros20 committed:
Added new community section, fixes for AppsCarousel

gabros20 committed:
Updated AppsCarousel

gabros20 committed:
Updated 3xl screen szie config

gabros20 committed:
Updated AppCarousel to show only 3 items

gabros20 committed:
Merge branch 'main' into feature/homepage-carousel

gabros20 committed:
fix: merge issues

gabros20 committed:
fix: layout

gabros20 committed:
feat: updated carousel and title

alex-beckett committed:
Merge pull request #12 from celestiaorg/feature/homepage-carousel

Feature/homepage carousel

gabros20 committed:
fix: carousel button position

alex-beckett committed:
Merge pull request #13 from celestiaorg/feature/homepage-carousel

fix: carousel button position

alex-beckett committed:
Add files via upload

alex-beckett committed:
Update apps-on-celestia.js

alex-beckett committed:
Update apps-on-celestia.js

gabros20 committed:
fix: increase padding around active carousel item

alex-beckett committed:
Merge pull request #14 from celestiaorg/feature/homepage-carousel

fix: increase padding around active carousel item

gabros20 committed:
fix: carousel shift to rigt due to scrollbar, mobile layout

gabros20 committed:
fix: padding on mobile

alex-beckett committed:
Merge pull request #15 from celestiaorg/feature/homepage-carousel

Feature/homepage carousel

gabros20 committed:
Merge branch 'main' into feature/homepage-carousel

gabros20 committed:
fix: remove CommunityCarousel from homepage

alex-beckett committed:
Merge pull request #16 from celestiaorg/feature/homepage-carousel

Feature/homepage carousel

alex-beckett committed:
Update apps-on-celestia.js

alex-beckett committed:
Add files via upload

alex-beckett committed:
Update apps-on-celestia.js

alex-beckett committed:
Update apps-on-celestia.js

alex-beckett committed:
Update apps-on-celestia.js

alex-beckett committed:
Add files via upload

alex-beckett committed:
Add files via upload

alex-beckett committed:
Update apps-on-celestia.js

alex-beckett committed:
Update apps-on-celestia.js

alex-beckett committed:
Update apps-on-celestia.js

alex-beckett committed:
Update apps-on-celestia.js

alex-beckett committed:
Update apps-on-celestia.js

alex-beckett committed:
Update apps-on-celestia.js

gabros20 committed:
feat: added gradient to heading text

alex-beckett committed:
Merge pull request #20 from celestiaorg/feature/homepage-carousel

feat: added gradient to heading text

alex-beckett committed:
Update apps-on-celestia.js

alex-beckett committed:
Add files via upload

alex-beckett committed:
Update apps-on-celestia.js

gabros20 committed:
fix: scroll speed change issue on mobile

alex-beckett committed:
Merge pull request #22 from celestiaorg/feature/homepage-carousel

fix: scroll speed change issue on mobile

alex-beckett committed:
Update apps-on-celestia.js

alex-beckett committed:
Add files via upload

gabros20 committed:
refactor: update AppsCarousel to use custom scrolling logic and remove Slider dependency

gabros20 committed:
feat: replace react-slick with custom GPU-accelerated marquee using Framer Motion

- Remove buggy react-slick dependency and CSS imports
- Implement custom infinite marquee with Framer Motion animations
- Add GPU acceleration with translate3d and optimized transforms
- Disable user interactions for pure auto-scroll experience
- Maintain responsive design (3 cards desktop, 1 mobile) with max-width caps
- Preserve all styling, gaps, and card link functionality
- Improve performance with delta-time based movement and hardware acceleration
Breaking: Requires framer-motion dependency

alex-beckett committed:
Merge pull request #23 from celestiaorg/feature/homepage-carousel

Feature/homepage carousel

gabros20 committed:
Merge branch 'main' into feature/homepage-carousel

alex-beckett committed:
Merge pull request #24 from celestiaorg/feature/homepage-carousel

Feature/homepage carousel

alex-beckett committed:
Delete public/images/app/homepage/apps-on-celestia/kinto.jpg

alex-beckett committed:
Delete public/images/app/homepage/apps-on-celestia/pay.jpg

alex-beckett committed:
Delete public/images/app/homepage/apps-on-celestia/leap.jpg

alex-beckett committed:
Delete public/images/app/homepage/apps-on-celestia/b3.jpg

alex-beckett committed:
Delete public/images/app/homepage/apps-on-celestia/derive.jpg

alex-beckett committed:
Delete public/images/app/homepage/apps-on-celestia/eclipse.jpg

alex-beckett committed:
Merge pull request #106 from alex-beckett/homepage-carousel-2

Update content
